src/Components/ActivationKeys/__tests__/ActivationKeys.test.js

```diff
@@ -65,6 +67,8 @@ jest.mock(
   () => () => <div>Unavailable</div>
 );
 
+useFeatureFlag.mockReturnValue(true);
```

Comment:
does this set the value to always true even if we turn off the feature flag one day?

Reply:
ignore me...this is a test file. byeeeee